configs/test/issue_trackers/config.yaml

```diff
@@ -19,7 +19,102 @@
 #   - %SANITIZER%: The sanitizers associated with the testcase.
 #   - %SEVERITY%: The security severity of the testcase.
 #   - %PLATFORM%: The platform the testcase was found on.
-test-project:
+test-project-jira:
```

Comment:
Are these changes required? 

I don't see any test changes that require this, and we typically wouldn't include them here either -- we prefer to inject these/mock this in the test setup instead.

Reply:
Added this as an example for other users. Renaming back test-project-monorail to test-project to avoid breaking any tests